NUOPC/hycom_nuopc_flags.F90

```diff
@@ -34,7 +34,18 @@ module hycom_nuopc_flags
     IMPORT_ERROR     = import_flag(-1), &
```

Comment:
I think it would be better to put comment/explanation about each of these new options.

Reply:
Done